automated/linux/signaltest/signaltest.sh

```diff
@@ -10,19 +10,19 @@ RESULT_FILE="${OUTPUT}/result.txt"
 
 PRIORITY="98"
 THREADS="2"
-LOOPS="10000"
 MAX_LATENCY="100"
+DURATION="1m"
 
 usage() {
-    echo "Usage: $0 [-p priority] [-t threads] [-l loops] [-m latency]" 1>&2
+    echo "Usage: $0 [-r runtime] [-p priority] [-t threads] [-l loops] [-m latency]" 1>&2
```

Comment:
I started to replace the loops argument in favor for duration but I was not sure if it's a good idea.

It certainly will break someones setup. There is no strong technical need to drop the loops argument but everyone has to define both variables. So in short I would like to avoid that both values are defined, e.g.

```
LOOPS: 1000
DURATION: 1m
```

One possible hack is do something like:
```
run:
  steps:
  - OPTARG=""; if [ ! -z "${LOOPS}" ] ; then OPTARG="-l ${LOOPS}"; fi
  -  ./signaltest.sh -D "${DURATION}" -p "${PRIORITY}" -t "${THREADS}" -m "${MAX_LATENCY}" "$OPTARG"
```
Comments?

Reply:
Another point to keep in mind.  This PR depends on an updated version of rt-tests (at least v1.4). I have seen that the tree contains sometimes binaries. I suppose those should be updated as well. Personally, I would not ship binaries here but I might lake some background why these binaries have been added in the first place.

Reply:
The backgroun is pretty simple. We can't build them on the target (in most cases). You can ship the binaries with rootfs in some cases but this prevents running the test in other envs. In the old days we tried packaged versions from OS (ubuntu) but since LKFT started with OE based build we moved to pre-built binaries in the repository. IMHO the best option now will be to replace the binaries with new version (compatible with changes you're making). @roxell any thoughts about that?

Reply:
I'm fine with that.

Reply:
BTW, I have started to add some of the missing rt-tests (ptsematest, svsematest, ...). Do you want me to add them to this PR or should I just wait until we resolved this PR and send an new PR?

Reply:
I think I just send a new PR for adding the missing rt-tests. It's probably better to finish up this PR first.